common/src/chain/tokens/mod.rs

```diff
@@ -0,0 +1,46 @@
+use parity_scale_codec::{Decode, Encode};
```

Comment:
Nitpick: the license header is missing. 🙃

Reply:
Thanks, but this is the draft pull request, and not ready for review yet. When it will be done, I mark this PR as "ready for review". Have a look on github UI, there is a special label where you can see a status. Btw, if you have any suggestions about tokens architecture, welcome to tell us here.